### DISCUSSION TITLE
Add support for Hazelcast to do session caching

### DISCUSSION THREAD

**Discussion by nvchaudhari1991** [OpenIDC/mod_auth_openidc#996] [Ideas]
Would it be possible to add the support for Hazelcast to do session caching? 

https://hazelcast.com/clients/cplusplus/

**Comment by zandbelt** [comment]
someone would have to write a C wrapper around the C++ client and link mod_auth_openidc against that; PRs are welcome, otherwise a commercial discussion via sales@openidc.com

**Comment by zandbelt** [comment]
based on another request we are looking into adding Hazelcast support as a commercial option; let us know at sales@openidc.com if you are interested
